### PR TITLE
Fix typing of fl>= and extfl>=

### PR DESCRIPTION
**Pull request by miraleung**
Addressing issue #115, based on behaviour for fl<=.

### CODE REVIEW

File: typed-racket-test/succeed/flonum-typecheck.rkt [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#lang typed/racket
+(require racket/flonum
+         typed/rackunit)
+
+
+;;Flonum typechecking tests
+;; -FlZero -Fl
+(check-eq?
```
Comment (shekari) [inline typed-racket-test/succeed/flonum-typecheck.rkt]: I'm confused at what these cases are testing. It seems like this issue was about filters but this is testing that in a very indirect manner, by making sure that the optimizer doesn't do dead code elimination on the wrong branch. Would these be better tested by the unit tests for typechecking?

Comment (miraleung) [inline typed-racket-test/succeed/flonum-typecheck.rkt]: Those tests are primarily aimed at regression testing the typechecks, but they can go if they are superfluous. I couldn't find a good way of "check-expecting" for type errors in the unit tests (check-exn doesn't catch those), hence the block at the end.

It looked to me like this issue concerned fl>= typechecks that shouldn't be passing, which this commit covers.

Comment (shekari) [inline typed-racket-test/succeed/flonum-typecheck.rkt]: I think the issue is that you were looking in the wrong file. This file is not a unit test (as we call it) but an integration test, as it checks the entire TR system e2e. The unittests for typechecking are located here: https://github.com/racket/typed-racket/blob/master/typed-racket-test/unit-tests/typecheck-tests.rkt. And those much more easily support the kind of test you are trying to write, and support negative tests.

**Comment by miraleung** [comment]
Since the issue was open, I didn't see that it had already been dealt with. Thanks for letting me know about the tests; now moved to the correct file in a separate pull request.
